### DISCUSSION TITLE
how to use the constellation_type-"custom"

### DISCUSSION THREAD

**Discussion by GCX-art** [NVlabs/sionna#666] [Q&A]
I noticed that there is a constellation_type-"custom" in the [constellation](https://nvlabs.github.io/sionna/api/mapping.html#constellation), but I don't know how to use it, could you give me a code snippet on how to use it.

**Comment by SebastianCa** [comment]
Hi @GCX-art,
you can find an example of custom constellations in the [end-to-end learning notebook](https://nvlabs.github.io/sionna/examples/Autoencoder.html#).
